src/lib/types.ts

```diff
@@ -203,73 +225,81 @@ export interface VolumeSnapshot {
   digest?: string
   id?: string
   size?: number
+  status?: string
 }
 
 export interface ApiDNSConfig {
   skip_registration?: boolean
 }
 
+/** A file that will be written to the Machine. One of RawValue or SecretName must be set. */
 export interface ApiFile {
   /**
    * GuestPath is the path on the machine where the file will be written and must be an absolute path.
-   * i.e. /full/path/to/file.json
+   * For example: /full/path/to/file.json
    */
   guest_path?: string
-  /** RawValue containts the base64 encoded string of the file contents. */
+  /** The base64 encoded string of the file contents. */
   raw_value?: string
-  /** SecretName is the name of the secret that contains the base64 encoded file contents. */
+  /** The name of the secret that contains the base64 encoded file contents. */
   secret_name?: string
 }
 
 export interface ApiHTTPOptions {
   compress?: boolean
+  h2_backend?: boolean
   response?: ApiHTTPResponseOptions
 }
 
 export interface ApiHTTPResponseOptions {
   headers?: Record<string, any>
 }
 
+/** An optional object that defines one or more named checks. The key for each check is the check name. */
 export interface ApiMachineCheck {
+  /** The time to wait after a VM starts before checking its health */
   grace_period?: string
   headers?: ApiMachineHTTPHeader[]
+  /** The time between connectivity checks */
   interval?: string
+  /** For http checks, the HTTP method to use to when making the request */
   method?: string
+  /** For http checks, the path to send the request to */
   path?: string
+  /** The port to connect to, often the same as internal_port */
   port?: number
+  /** For http checks, whether to use http or https */
   protocol?: string
+  /** The maximum time a connection can take before being reported as failing its health check */
   timeout?: string
+  /** If the protocol is https, the hostname to use for TLS certificate validation */
   tls_server_name?: string
+  /** For http checks with https protocol, whether or not to verify the TLS certificate */
   tls_skip_verify?: boolean
+  /** tcp or http */
   type?: string
 }
 
 export interface ApiMachineConfig {
+  /** Optional boolean telling the Machine to destroy itself once it’s complete (default false) */
   auto_destroy?: boolean
   checks?: Record<string, ApiMachineCheck>
   /** Deprecated: use Service.Autostart instead */
   disable_machine_autostart?: boolean
   dns?: ApiDNSConfig
-  /**
-   * Fields managed from fly.toml
-   * If you add anything here, ensure appconfig.Config.ToMachine() is updated
-   */
+  /** An object filled with key/value pairs to be set as environment variables */
   env?: Record<string, string>
   files?: ApiFile[]
   guest?: ApiMachineGuest
-  host_dedication_id?: string
```

Comment:
Deleting this field would be a breaking change since it's an exported interface.

Let's mark it as deprecated instead?